SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -15,14 +15,12 @@
 
 namespace SonarQube.TeamBuild.PreProcessor
 {
-    public sealed class SonarWebService : IDisposable
```

Comment:
From a design perspective, I think it is cleaner to not dispose an object that you did not instantiate yourself - to allow it to be shared for example.